mmv1/templates/terraform/examples/firebase_extentions_instance_resize_image.tf.erb

```diff
@@ -0,0 +1,63 @@
+resource "google_storage_bucket" "images" {
+  provider                    = google-beta
+  project                     = "<%= ctx[:test_env_vars]['project_id'] %>"
+  name                        = "<%= ctx[:vars]['bucket_id'] %>"
+  location                    = "US"
+  uniform_bucket_level_access = true
+
+  # Delete all objects when the bucket is deleted
+  force_destroy = true
+}
+
+resource "google_service_account" "resize_image" {
+  provider     = google-beta
+  project      = "<%= ctx[:test_env_vars]['project_id'] %>"
+  account_id   = "<%= ctx[:vars]['service-account-id'] %>"
+}
+
+resource "google_project_iam_member" "resize_image" {
+  provider = google-beta
+  project  = "<%= ctx[:test_env_vars]['project_id'] %>"
+  role     = "roles/datastore.user"
+  member   = "serviceAccount:${google_service_account.resize_image.email}"
+}
+
+resource "google_firebase_extensions_instance" "resize_image" {
+  provider = google-beta
+  project  = "<%= ctx[:test_env_vars]['project_id'] %>"
+  instance_id = "<%= ctx[:vars]['instance-id'] %>"
+  service_account_email = google_service_account.resize_image.email
```

Comment:
Providing a custom `service_account_email` is currently considered a "legacy" feature, which we might deprecate in the future... I'm thinking we should support it as "output only" in TF.

Reply:
Okay!